src/txdb.cpp

```diff
@@ -240,6 +240,10 @@ bool CBlockTreeDB::WriteBatchSync(const std::vector<std::pair<int, const CBlockF
     return WriteBatch(batch, true);
 }
 
+bool CBlockTreeDB::HasTxIndex(const uint256& txid) {
```

Comment:
I understand that you are trying to replicate surrounding code, but when adding new code we should follow code guidelines (ie move the `{` down to the next line)